doc/nobol/src/nobol7.lalrpop

```diff
@@ -0,0 +1,10 @@
+use super::{Var, Lit, Stmt};
+
+grammar;
+
+pub Var: Var = <r"[a-z]"> => <>.chars().next().unwrap().into();
+
+pub Lit: Lit = <l:r#""(\\\\|\\"|[^"])*""#> =>
```

Comment:
Shouldn't this be `"(\\\\|\\"|[^"\\])*"` (exclude `\` too)?

Reply:
Probably! Let me look into that, thanks!